### PR TITLE
Update mac-installs.md

### PR DESCRIPTION
**Pull request by TrinityChristiana**
Updating installations with error prevention.

### CODE REVIEW

File: src/data/curriculum/python/week-00/installations/mac-installs.md [hunk 1]
```diff
@@ -66,11 +66,17 @@ There are a few lines to add to the .zshrc file before we can use `pyenv`
 1. Save and exit VS Code
 1. Back in the terminal, load the changes to the file: `source ~/.zshrc`
 
+### Update xcode
```
Comment (TrinityChristiana) [inline src/data/curriculum/python/week-00/installations/mac-installs.md:69]: This was causing an issue with installing pipenv, so I have them just go ahead and make sure they have the most up-to-date version.

https://github.com/orgs/nss-evening-web-development/discussions/206
https://github.com/orgs/nss-evening-web-development/discussions/203

File: src/data/curriculum/python/week-00/installations/mac-installs.md [hunk 2]
```diff
@@ -66,11 +66,17 @@ There are a few lines to add to the .zshrc file before we can use `pyenv`
...
+2. if you get the following error `xcode-select: error: command line tools are already installed, use "Software Update" to install updates` then run tese commands.
+    -  `sudo rm -rf /Library/Developer/CommandLineTools` This will delete your old version of xcode
+    -  `xcode-select --install` This will reinstall xcode at an updated version
+
 ### Download Python
 
 1. In the same terminal window, download the version of Python: `pyenv install 3.9.10`
 2. After the download finishes, set that version of Python to the global version: `pyenv global 3.9.10`
-3. Now your computer’s Python version should be 3.9.10. Check it with `python3 --version`. **If something other than 3.9.10 comes back, reach out to an instructor now.**
+3. Now your computer’s Python version should be 3.9.10. Check it with `python --version`. **If something other than 3.9.10 comes back, reach out to an instructor now.**
```
Comment (TrinityChristiana) [inline src/data/curriculum/python/week-00/installations/mac-installs.md:79]: `python3` is not being updated with these installations. Changing this to just `python`

File: src/data/curriculum/python/week-00/installations/mac-installs.md [hunk 3]
```diff
@@ -82,10 +88,10 @@ In some ways, `pipenv` does the same thing as `node_modules` and `package.json`.
 1. Install the package globally:
 
     ```sh
-    python3 -m pip install pipenv
+    python -m pip install pipenv
```
Comment (TrinityChristiana) [inline src/data/curriculum/python/week-00/installations/mac-installs.md:91]: `python3` is not being updated with these installations. Changing this to just `python`

File: src/data/curriculum/python/week-00/installations/mac-installs.md [hunk 4]
```diff
@@ -82,10 +88,10 @@ In some ways, `pipenv` does the same thing as `node_modules` and `package.json`.
...
     ```
 
-1. Check that the package is usable: `pipenv --version`
+1. Check that the package is usable: `pipenv --version` should output a version of pipenv. Something like: `pipenv, version 2022.9.24`
```
Comment (TrinityChristiana) [inline src/data/curriculum/python/week-00/installations/mac-installs.md:94]: Updated the description to help with knowing when this is working correctly.

File: src/data/curriculum/python/week-00/installations/mac-installs.md (hunk 3 above)
Comment (TrinityChristiana) [inline src/data/curriculum/python/week-00/installations/mac-installs.md:91]:
```suggestion
    sudo python -m pip install pipenv
```

**Comment by drteresavasquez** [comment]
@TrinityChristiana let me know when you’re ready for a review.

**Comment by TrinityChristiana** [comment]
> @TrinityChristiana let me know when you’re ready for a review.

This is now ready fro review!

**Review by drteresavasquez — APPROVED** [review]
👍🏼

**Pull request merged**